google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -165,8 +166,18 @@ def list_findings(self, source_id):
           Returns:
               object: An API response containing all the CSCC findings.
         """
-        response = self.repository.findings.list(parent=source_id)
-        return response
+
+        try:
+            LOGGER.debug('Listing findings.')
+
+            response = self.repository.findings.list(parent=source_id,
+                                                     filter='state="ACTIVE"',
+                                                     pageSize=100)
```

Comment:
Any reason why the page size is 100?

Reply:
Page size can be anywhere between 1 and 1000. It's the default (10) right now. I have bumped it to 100 as it seems to be higher than we we had (10) but not too high.

Reply:
Any reasons to not maxing it out at 1000? is there any disadvantage to do so compare to say 100?

Reply:
It can be anywhere between 10 and 1000 based on need/preference.

Reply:
I would change this to 1000 if there isnt any disadvantage of doing so.

Reply:
Updated to 1000.